### PR TITLE
fix nep11 tracker

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Close https://github.com/neo-project/neo-modules/issues/754

### CODE REVIEW

File: src/TokensTracker/Trackers/TrackerBase.cs [hunk 1]
```diff
@@ -129,7 +129,8 @@ protected TransferRecord GetTransferRecord(UInt160 asset, Array stateItems)
             return stateItems.Count switch
             {
                 3 => new TransferRecord(asset, @from, to, null, amountItem.GetInteger()),
-                4 when (stateItems[3] is ByteString tokenId) => new TransferRecord(asset, @from, to, tokenId, amountItem.GetInteger()),
+                4 when stateItems[3] is ByteString tokenId => new TransferRecord(asset, @from, to, tokenId, amountItem.GetInteger()),
+                4 when stateItems[3] is Integer tokenId => new TransferRecord(asset, from, to, tokenId.GetSpan().ToArray(), amountItem.GetInteger()),
```
Comment (erikzhang) [inline src/TokensTracker/Trackers/TrackerBase.cs:133]: Why `tokenId` is an `Integer`?

Comment (roman-khimov) [reply src/TokensTracker/Trackers/TrackerBase.cs:133]: Because it can be used in absence of neo-project/neo#1879?

Comment (erikzhang) [reply src/TokensTracker/Trackers/TrackerBase.cs:133]: neo-project/neo#2810

Comment (vikkkko) [reply src/TokensTracker/Trackers/TrackerBase.cs:133]: When the developer calls the contract, the tokenid is used as an Integer. The notified field is Integer。

Contract Example：
`
public class Test : SmartContract
{
    [DisplayName("Test")]
    public static event Action<ByteString> OnTest;

    public static void CallEvent(ByteString tokenId)
    {
        OnTest(tokenId);
    }
}
`

`
sb.EmitDynamicCall(contract, "callEvent", 1);
`
type of event :  Integer

`
sb.EmitDynamicCall(contract, "callEvent", ((BigInteger)1).ToByteArray());
`
type of event :  ByteString

Comment (vikkkko) [reply src/TokensTracker/Trackers/TrackerBase.cs:133]: When the developer calls the contract, the tokenid is used as an Integer. The notified field is Integer。

event Example：
```
public static event Action<ByteString> OnTest;

public static void CallEvent(ByteString tokenId)
{
        OnTest(tokenId);
}
```

`
sb.EmitDynamicCall(contract, "callEvent", 1);
`
type of the field  in event :  Integer


`
sb.EmitDynamicCall(contract, "callEvent", ((BigInteger)1).ToByteArray());
`
type of the field in event:  ByteString

Comment (Ashuaidehao) [reply src/TokensTracker/Trackers/TrackerBase.cs:133]: > When the developer calls the contract, the tokenid is used as an Integer. The notified field is Integer。
> 
> event Example：
> 
> ```
> public static event Action<ByteString> OnTest;
> 
> public static void CallEvent(ByteString tokenId)
> {
>         OnTest(tokenId);
> }
> ```
> 
> `sb.EmitDynamicCall(contract, "callEvent", 1);` type of the field in event : Integer
> 
> `sb.EmitDynamicCall(contract, "callEvent", ((BigInteger)1).ToByteArray());` type of the field in event: ByteString


For https://github.com/neo-project/neo/pull/2810, we can ignore these "invalid" transactions and close this PR, right?

**Pull request closed without merging**